source/custom/nixnet_converters.h

```diff
@@ -13,6 +13,7 @@ void SetCanCommResponse(const u32 input, nixnet_grpc::CanCommResponse* output);
 void SetFlexRayCommResponse(const u32 input, nixnet_grpc::FlexRayCommResponse* output);
 void SetLinCommResponse(const u32* input, nixnet_grpc::LinCommResponse* output);
 void SetSessionInfoResponse(const u32& input, nixnet_grpc::SessionInfoResponse* output);
+void convert_enet_frame_to_grpc(const void* input, nixnet_grpc::FrameBufferResponse* output, u32 protocol, std::map<std::int32_t, std::int32_t> enetflags_output_map);
```

Comment:
You don't need this change now, since you call wrapper API around this from test.